scripts/createFeed.mjs

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import glob from 'glob';
+import fs from 'fs';
+import xml from "xml";
+import process from 'process';
+import {parseArgs} from 'node:util';
+
+let options = {
+  releases: {
+    type: 'string'
+  },
+  blog: {
+    type: 'string'
+  }
+};
+
+(async function createRssFeed() {
+
+  let args = parseArgs({options, allowPositionals: true});
+
+  if (args.positionals.length < 1) {
+    console.error('Expected at least one argument');
+    process.exit(1);
+  }
+
+  let type = args.positionals[0];
+  if (type !== 'releases' && type !== 'blog') {
+    console.error('Expected argument to be either releases or blog');
+    process.exit(1);
+  }
+
+  let posts = getFeed(type);
+  const feedObject = {
+    rss: [
+      {_attr: {
+          version: "2.0",
+          "xmlns:atom": "http://www.w3.org/2005/Atom",
+        },
+      },
+      {channel: [
+          {"atom:link": {
+              _attr: {
+                href: `https://react-spectrum.adobe.com/${type}/${type}-feed.rss`,
+                rel: "self",
+                type: "application/rss+xml",
+              },
+            },
+          },
+          {title: "Adobe React Spectrum"},
+          {link: `https://react-spectrum.adobe.com/`},
+          {description: "A collection of libraries and tools that help you build adaptive, accessible, and robust user experiences."},
```

Comment:
Should the details here be made more specific to "releases"/"blog posts"? At the moment when I subscribe to these feed via an aggregator, they both come up as "Adobe React Spectrum". Alternatively, perhaps they both should be merged under a single feed so you get both the release and blog posts?

Reply:
yeah i can update it so that it is more specific. i thought about maybe doing it under a single feed but figured some people might not want to get blog posts (or vice versa) and this way, they could choose what they wanted to subscribe to